### PR TITLE
[PBIOS-456] Progress Simple Kit

### PR DESCRIPTION
**Pull request by RachelRadford21**
**What does this PR do?** 
[PBIOS-456] Progress Simple Kit

**Screenshots:** 
Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 16 38 16

### Checklist
- [x] **LABELS** - Add a label: `breaking`, `bug`, `improvement`, `documentation`, or `enhancement`. See [Labels](https://github.com/powerhome/playbook-apple/labels) for descriptions.
- [x] **RELEASES** - Add the appropriate label: `Ready for Testing` / `Ready for Release`
- [x] **TESTING** - Have you tested your story?

### CODE REVIEW

File: Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  let progressColor: Color
+  let variant: Variant
+  
+  public init(
+    progress: Binding<Double> = .constant(0.0),
+    value: Binding<Int> = .constant(2),
+    maxValue: Int = 10,
+    progressColor: Color = .pbPrimary,
+    variant: Variant = .default
+
```
Comment (isismsilva) [inline Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift:25]: remove extra empty spaces

File: Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  
+  var variantView: some View {
+    HStack {
+      switch variant {
+      case .default: ProgressView(value: progress, total: 1)
+      case .settingValue: ProgressView(value: CGFloat(value), total: CGFloat(maxValue))
+      }
+    }
+    .tint(progressColor)
+
```
Comment (isismsilva) [inline Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift:52]: remove extra empty spaces

File: Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+public extension PBProgressSimple {
+  enum Variant {
+    case `default`, settingValue
+  }
+  
+  var variantView: some View {
+    HStack {
+      switch variant {
+      case .default: ProgressView(value: progress, total: 1)
```
Comment (isismsilva) [inline Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift:47]: Why is 1 the total value? Is it right?

**Review by isismsilva — CHANGES_REQUESTED** [review]

File: Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift (hunk 3 above)
Comment (RachelRadford21) [reply Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift:46]: Yes, it's a double value so 1 is essentially "up to" 100%.

Comment (RachelRadford21) [reply Sources/Playbook/Components/Progress Simple/PBProgressSimple.swift:46]: idk if this makes sense but maybe another way to say is that its calculating the progress value by what the total allowed is. so if this tracks up to 100% then the value provided in the catalog will be incremented based on the total.

**Review by isismsilva — APPROVED** [review]

**Pull request merged**